iOS/MMIWG2S/Utilities/MmiwUtility.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  MmiwUtility.swift
+//  MMIWG2S
+//
+//  Created by Stephen Emery on 7/12/21.
+//  Copyright © 2021 Google LLC. All rights reserved.
+//
+
+import UIKit
+import ARKit
+
+class MmiwUtility {
+    static var faceViewController: UIViewController {
```

Comment:
shouldn't this be a lazy var so we aren't instantiating it more than once? Or is that the intent, it should probably be a function (`createFaceViewController()`).

Reply:
I didn't realize that the `lazy` property kept things from instantiating twice. I added lazy and saw this: `'lazy' must not be used on an already-lazy global`. Apparently static variables are always lazily loaded anyway. Looks like we are good.